Submissions/@ushicode.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Collins Ushi",
+    "img": "https://avatars.githubusercontent.com/u/4209868?v=4",
+    "email": "collinsushi@yahoo.com",
+    "links": {
+      "website": "https://github.com/ushicode",
+      "linkedin": "https://www.linkedin.com/in/collinsushi",
+      "github": "https://github.com/ushicode>"
```

Comment:
please remove the >

Reply:
Changes made; the > has been removed from the code